src/LightningInspectorInjectedScript.js

```diff
@@ -170,6 +170,18 @@ function AuraInspector() {
             console.error('Lightning Inspector: Error initializing page hooks for transactions', e);
         }
 
+        try {
+            bootstrapPerfDevTools.call(this);
```

Comment:
This got deleted which breaks the performance panel. I want to kill it, but not having it work is a bad look. So at least making it work for now.